frontend/javascript/components/docupload/image-document-pages-sortable.vue

```diff
@@ -0,0 +1,51 @@
+<script setup>
+/* This is a separate component because we need to provide Vue3 composition setup for useSortable,
+which looks like the most reasonable replacement for vuedraggable (Vue2 only);
+other alternatives did not work (vuedraggable-next and a different thin SortableJS wrapper). */
+
+import { useSortable } from '@vueuse/integrations/useSortable'
+import { ref, defineEmits, computed } from 'vue'
+import ImagePage from './image-page.vue'
+
+const props = defineProps({ pages: Array })
+
+// once updated to Vue>=3.4
```

Comment:
we could also upgrade right now! :)

Reply:
I can give it a quick try - I've glanced at 3.2→3.3→3.4 changelogs/deprecations and they seemed fairly benign. But I feel like I would have to test *all* Vue code to avoid subtle breakage like the situation here...